src/TraceEvent/Computers/SampleProfilerThreadTimeComputer.cs

```diff
@@ -202,17 +209,20 @@ public void GenerateThreadTimeStacks(MutableTraceEventStackSource outputStackSou
 
                 StackSourceCallStackIndex stackIndex = GetCallStack(data, thread);
 
-                // Tack on additional info about the event. 
```

Comment:
I think you can put the 'TraceEventSource' Boolean around this whole 
```
eventSource.Dynamic.All += .delegate (TraceEvent data)  {
...
}
```
statement above (you can also rip out the TODO FIX NOW line).   It is OK to lose the LogThreadStack update that disappears.   This is what the code looked like before the feature was added.